### PR TITLE
Add bug for kernel-tests

### PR DESCRIPTION
**Pull request by rodrigc**

### CODE REVIEW

File: data/fedora-update.yaml [hunk 1]
```diff
@@ -45,13 +45,12 @@ kernel-tests:
     status: in-progress
     links:
         repo: https://git.fedorahosted.org/cgit/kernel-tests.git
+        bug: https://bugzilla.redhat.com/show_bug.cgi?id=1282165
```
Comment (encukou) [inline data/fedora-update.yaml]: That's the wrong bug; I changed it to https://bugzilla.redhat.com/show_bug.cgi?id=1293015

**Comment by encukou** [comment]
Thanks!

**Comment by rodrigc** [comment]
Thanks!  Who can commit the patches I submitted in https://bugzilla.redhat.com/show_bug.cgi?id=1293015 ?

At https://git.fedorahosted.org/git/kernel-tests.git , Zbigniew Jędrzejewski-Szmek and Justin Forbes were the last two people who committed code.

**Comment by encukou** [comment]
It's probably best if you ask them. You can say you "Need additional information from" one of them in the Bugzilla.

**Pull request merged**